c2corg_ui/static/js/documentediting.js

```diff
@@ -74,6 +75,12 @@ app.DocumentEditingController = function($scope, $element, $attrs, gettextCatalo
   this.ngeoLocation_ = ngeoLocation;
 
   /**
+   * @type {angular.$cookies}
+   * @private
+   */
+  this.cookies_ = $cookies;
```

Comment:
This property is not needed since used in the constructor => you can use ``$cookies`` directly.